CAIPs/caip-294.md

```diff
@@ -497,11 +502,13 @@ TODO
 - [CAIP-27][caip-27] - Blockchain ID Specification
 - [CAIP-25][caip-25] - Blockchain ID Specification
 - [CAIP-282][caip-282] - Browser Wallet Discovery Interface
+- [CAIP-341][caip-341] - Extension ID Target Type Specification
 
 [eip-6963]: https://eips.ethereum.org/EIPS/eip-6963
 [caip-27]: https://chainagnostic.org/CAIPs/caip-27
 [caip-25]: https://chainagnostic.org/CAIPs/caip-25
 [caip-282]: https://chainagnostic.org/CAIPs/caip-282
+[caip-341]: https://github.com/ChainAgnostic/CAIPs/blob/656551f800843b92243fb08ca6c24e805ad149a3/CAIPs/caip-341.md
```

Comment:
```suggestion
- [CAIP-341][caip-341] - Extension ID Target Type Specification
- [externally_connectable][externally_connectable] - Chrome's `externally_connectable` browser API documentation

[eip-6963]: https://eips.ethereum.org/EIPS/eip-6963
[caip-27]: https://chainagnostic.org/CAIPs/caip-27
[caip-25]: https://chainagnostic.org/CAIPs/caip-25
[caip-282]: https://chainagnostic.org/CAIPs/caip-282
[caip-341]: https://github.com/ChainAgnostic/CAIPs/blob/656551f800843b92243fb08ca6c24e805ad149a3/CAIPs/caip-341.md
[externally_connectable]: https://developer.chrome.com/docs/extensions/reference/manifest/externally-connectable
```

Reply:
resolved in [c06811b](https://github.com/pedrouid/CAIPs/pull/9/commits/c06811beb1faca27c936c7232220625fc004ffdb)